components/02-molecules/article&blog-post-elements/author-bio/author-bio-short.html

```diff
@@ -1,28 +1,28 @@
-<div class="list-teaser__footer">
-    <div class="author-bio author-bio--short" itemscope itemtype="http://schema.org/Person">
+<footer class="list-teaser__footer">
+    <article class="author-bio author-bio--short" itemscope itemtype="http://schema.org/Person">
         <div class="author-bio__image avatar avatar--small">
             <a class="author-bio__anchor avatar__link" href="#author-bio">
                 <img itemprop="photo" class="avatar__image" src={context.app.uri("assets/example-content/heribert.jpg")} alt="Portrait von Heribert Innoq" />
             </a>
         </div>
-        <div class="author-bio__text">
-            <div class="author-bio__name" itemprop="name">
+        <p class="author-bio__text">
+            <h1 class="author-bio__name" itemprop="name">
                 <a class="author-bio__link" itemprop="url" href="#">Heribert Innoq</a>
-            </div>
-            <div class="author-bio__info" itemprop="jobTitle" >Senior Consultant</div>
-        </div>
-    </div>
-    <div class="author-bio author-bio--short" itemscope itemtype="http://schema.org/Person">
+            </h1>
+            <h2 class="author-bio__info" itemprop="jobTitle" >Senior Consultant</h2>
```

Comment:
s.o. Heading Tags würde ich in diesem Zusammenhang nicht verwenden. Dann eher ein `<span>` oder vll gibt es ja auhc noch etwas besseres. :)